frameworks/g-cloud-12/messages/homepage-sidebar.yml

```diff
@@ -1,5 +1,5 @@
 coming:
-  heading: 'G-Cloud&nbsp;12 is opening in March&nbsp;2525'
+  heading: 'G-Cloud&nbsp;12 is opening in March&nbsp;2020'
```

Comment:
May I suggest: G-Cloud 12 will open on 3 March 2020.

Reply:
👍 (I think we've kept the 'coming' date deliberately vague in the past to allow for last minute changes - no need to now that the date is confirmed & published).